github/actions_workflows.go

```diff
@@ -76,6 +76,15 @@ type WorkflowsPermissionsOpt struct {
 	RequireApprovalForForkPRWorkflows *bool `json:"require_approval_for_fork_pr_workflows,omitempty"`
 }
 
+// WorkflowApprovalPolicy represents the approval policy for workflows in a repository.
+type WorkflowApprovalPolicy struct {
```

Comment:
Seems like we can think of a better name for the struct.

This struct is returned from the following functions:

- GetForkPRContributorApprovalPermissionsInEnterprise
- GetForkPRContributorApprovalPermissionsInOrganization
- GetForkPRContributorApprovalPermissions

And is used as a parameter in:

- UpdateForkPRContributorApprovalPermissionsInEnterprise
- UpdateForkPRContributorApprovalPermissionsInOrganization
- UpdateForkPRContributorApprovalPermissions

What about:

```suggestion
type ContributorApprovalPermissions struct {
```

?